### PR TITLE
Dockerfile: update to github-pages 198, and some other fixes/improvements

### PR DESCRIPTION
**Pull request by thaJeztah**
See individual commits for details

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,9 +24,7 @@ ARG ENABLE_ARCHIVES=true
 ###
 # Set up base stages for building and deploying
 ###
-
-# Get basic configs and Jekyll env
-FROM docs/docker.github.io:docs-builder AS builderbase
+FROM starefossen/github-pages:198 AS builderbase
```
Comment (thaJeztah) [inline Dockerfile]: This (at least for now) cuts out our custom image, which was based on `starefossen/github-pages`, with some other changes that we currently don't use)

**Comment by thaJeztah** [comment]
@usha-mandya @StefanScherer ptal

File: Dockerfile (hunk 1 above)
Comment (StefanScherer) [reply Dockerfile]: Good idea to remove complexity!

Comment (thaJeztah) [reply Dockerfile]: The idea of the custom image was that we could update or do fixes in that image, and rebuild the archived docs, without having to update each of those branches.

If we stop publishing those archives that may no longer be relevant

**Comment by usha-mandya** [comment]
Thank you @thaJeztah.

**Pull request merged**